libcodechecker/server/session_manager.py

```diff
@@ -323,57 +314,34 @@ def __try_auth_ldap(self, auth_string):
     def get_user_name(auth_string):
         return auth_string.split(':')[0]
 
-    def _fetch_session_or_token(self, persistency_hash):
+    def get_db_session_token(self, user_name):
```

Comment:
We discussed that it will be possible later that multiple tokens belong to one user.
Shouldn't we prepare for that now?